classes/game.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'item'
+
+class Game < Item
+  def initialize(publish_date, multiplayer, last_played_at)
+    super(publish_date)
+    @multiplayer = multiplayer
+    @last_played_at = DateTime.parse(last_played_at)
+  end
+
+  def last_played_old
+    return true if @last_played_at < DateTime.now.next_year(-2)
+
+    false
```

Comment:
Hi @TimmyChan99 👋 ,
Good job so far, 
But I would suggest you remove the  redundant false on line 13 as line 11 will auto return false if the condition is not true.

Reply:
Thank you. Changes implemented.